harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -919,6 +919,51 @@ def checkpoint_and_restore(
 
         return (training_metrics["A"], training_metrics["B"])
 
+    def checkpoint_and_restore_no_callbacks(
```

Comment:
I don't think I understand what "_no_callbacks" means in the name here?

Reply:
there is a checkpoint_and_restore function that uses callback methods that are tied to pytorch_onevar_model. Wanted to make it clear in the code why there needs to be a second function, but I should probably add that as a dev note instead.